models/Products/product.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+
+const uuid = require('uuid/v4');
+const DataModel = require('../file-data-model.js');
+
+class Products extends DataModel {
+  constructor() {
+    super();
+    console.log()
+    this.schema = {
```

Comment:
from line 11-15 put outside of the class and call it schema. In the parentheses put schema in their.